bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -362,4 +367,18 @@ public String getName() {
         return FormContainer.super.getName();
     }
 
+    @JsonIgnore
```

Comment:
jsonignore should not be required in impl, I guess.

Reply:
I see it being used in other methods above of the same class, and so maybe it is?
cc: @rismehta